src/nft/components/profile/list/ListingButton.tsx

```diff
@@ -0,0 +1,144 @@
+import { Plural, t, Trans } from '@lingui/macro'
+import { BaseButton } from 'components/Button'
+import ms from 'ms.macro'
+import { BelowFloorWarningModal } from 'nft/components/profile/list/Modal/BelowFloorWarningModal'
+import { useIsMobile, useNFTList, useSellAsset } from 'nft/hooks'
+import { Listing, ListingStatus, WalletAsset } from 'nft/types'
+import { useEffect, useMemo, useState } from 'react'
+import styled from 'styled-components/macro'
+import { BREAKPOINTS } from 'theme'
+import shallow from 'zustand/shallow'
+
+import { getListings } from '../../bag/profile/utils'
+
+const BELOW_FLOOR_PRICE_THRESHOLD = 0.8
+
+const StyledListingButton = styled(BaseButton)<{ showResolveIssues: boolean; missingPrices: boolean }>`
+  background: ${({ showResolveIssues, theme }) => (showResolveIssues ? theme.accentFailure : theme.accentAction)};
+  color: ${({ theme }) => theme.accentTextLightPrimary};
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 24px;
+  padding: 16px;
+  border-radius: 12px;
+  width: min-content;
+  border: none;
+  cursor: ${({ missingPrices }) => (missingPrices ? 'auto' : 'pointer')};
+  opacity: ${({ showResolveIssues, missingPrices }) => (showResolveIssues ? '1' : missingPrices && '0.3')};
```

Comment:
really nitty nit: should either be `(showResolveIssues ? '1' : missingPrices ? '0.3' : '1')` or `!showResolveIssues && missingPrices && '0.3')` not a huge fan of mixing different types of logical operator styles